Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -39,18 +39,13 @@ public protocol ManifestResourceProvider {
     /// The path of the library resources.
     var libDir: AbsolutePath { get }
 
+    /// The path to the PackageDescription library.
+    var packageDescriptionDir: AbsolutePath { get }
```

Comment:
This was supposed to be a rename to `libDir`, we shouldn't have both. 
Anyway, the rename is not required because I separated out the implementation object for ManifestResourceProvider in UserToolchain, so this can be untouched.